### PR TITLE
Implementing Stack Using Queues

### PR DESCRIPTION
**Pull request by sricharan200**
Implementing stack using queues involves two queue data structures to show the Last-In-First-Out (LIFO) behavior of a stack. This implementation utilizes two methods: push, which adds an element to the top of the stack, and pop, which removes the top element.

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.
- [x] All new code is formatted with `clang-format -i --style=file path/to/your/file.java`

Note: I have raised a pull request. This is part of my coursework. Kindly, look into it. Thank you.

### CODE REVIEW

**Review by siriak — CHANGES_REQUESTED** [review]
Could you please replace main with some JUnit tests? (see https://github.com/TheAlgorithms/Java/tree/master/src/test/java/com/thealgorithms)

**Comment by sricharan200** [comment]
> Could you please replace main with some JUnit tests? (see https://github.com/TheAlgorithms/Java/tree/master/src/test/java/com/thealgorithms)

Hi, do you want me to push the code to the test folder instead of the main folder?

Correct me if I am wrong

**Comment by sricharan200** [comment]
> Could you please replace main with some JUnit tests? (see https://github.com/TheAlgorithms/Java/tree/master/src/test/java/com/thealgorithms)

I have made the required changes. Kindly, look into it. Thank you.

**Review by siriak — CHANGES_REQUESTED** [review]
Please review your submission. The implementation must be in StackUsingQueues.java and tests must be in StackUsingQueuesTests.java in tests folder. Also see PR checks that fail for details

**Comment by sricharan200** [comment]
> Please review your submission. The implementation must be in StackUsingQueues.java and tests must be in StackUsingQueuesTests.java in tests folder. Also see PR checks that fail for details

I have made necessary changes. Kindly, look into it. Thank you

File: src/main/java/com/thealgorithms/stacks/StackUsingQueues.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    public boolean empty() {
+        return queue2.isEmpty();
+    }
+
+    /**
+     * Main method to demonstrate stack operations.
+     * 
+     * @param args Command line arguments.
+     */
+    public static void main(String[] args) {
```
Comment (siriak) [inline src/main/java/com/thealgorithms/stacks/StackUsingQueues.java:77]: Please remove `main`, we don't need it. Tests already test the implementation

**Review by siriak — CHANGES_REQUESTED** [review]
Please add a test with more items in stack (at least 5) and fix PR checks, everything else looks good
image

**Comment by sricharan200** [comment]
Hey, I have made the changes. Please look into it. Thank you.

**Review by BamaCharanChhandogi — CHANGES_REQUESTED** [review]
There was a small 'clang' issue. Please solve it.
Your test cases are failing. Try to resolve the issue in your test cases

**Pull request closed without merging**